src/js/veteran-id-card/components/RequiredVeteranView.jsx

```diff
@@ -16,7 +18,9 @@ class RequiredVeteranView extends React.Component {
       // in our system.
       view = <SystemDownView messageLine1="We’re sorry. We can’t proceed with your request for a Veteran ID card because we can't confirm your military history right now. Please try again in a few minutes." messageLine2="If it still doesn’t work, please call the Vets.gov Help Desk at 1-855-574-7286, TTY: 1-800-877-8339. We’re here Monday &#8211; Friday, 8:00 a.m. &#8211; 8:00 p.m. (ET)."/>;
     } else if (this.props.userProfile.veteranStatus === 'OK') {
-      if (!serviceAvailable) {
+      if (serviceRateLimited) {
```

Comment:
Should we put this check below the `!serviceAvailable` one? Might as well tell the people aren't eligible now, since we already know.

Reply:
Ah, good point - I'll make an update.

Reply:
this update would conflict with @robertfairhead's updates analytics in https://github.com/department-of-veterans-affairs/vets-website/pull/6884. I'm going to wait for that to get pushed and provide a compatible update in a new PR afterward. Sorry, missed the comment before the PR went in.